VirtoCommerce.Storefront.IntegrationTests/Cart/ApiCartControllerTests.cs

```diff
@@ -74,22 +78,61 @@ public async Task GetCart_IfCartExistForUser_ShouldReturnExistedCart()
             //arrange
             _client
                 .Login("admin", "store")
-                .СlearCart()
-                .InsertCartItem(new AddCartItem { Id = "9cbd8f316e254a679ba34a900fccb076", Quantity = 1 });
+                .ClearCart()
+                .InsertCartItem(new AddCartItem { Id = Product.Quadcopter, Quantity = 1 });
 
             //act
-            var result = await GetCart(_client);
+            var result = await _client.GetCart();
 
             //assert
             GetCartComparationResult(
```

Comment:
Expected string to be <null>, but found "{
      "isAnonymous": [
        true,
        false
      ],
      "customerName": [
        "Anonymous",
        "admin"
      ],
      "type": [
        "cart",
        0,
        0
      ]
    }"